Default config does not report diffs, even though `--diff` appears to be executed

Docs: https://pycqa.github.io/isort/docs/configuration/github_action.html and https://github.com/isort/isort-action/blob/master/README.md say:

> `isort` configuration options to pass to the `isort` CLI. Defaults to `--check-only --diff`.

## Expected behaviour:

Output including diffs, similar to what I get running `isort --check-only --diff .` at CLI, e.g.

```
isort --check-only --diff .
ERROR: C:\Users\[REDACTED]\code\elliot-100_Spond\tests\test_spond.py Imports are incorrectly sorted and/or formatted.
--- C:\Users\[REDACTED]\code\elliot-100_Spond\tests\test_spond.py:before        2024-05-22 10:26:37.165779
+++ C:\Users\[REDACTED]\code\elliot-100_Spond\tests\test_spond.py:after 2024-05-22 10:59:23.726940
@@ -1,7 +1,6 @@
 """Test suite for Spond class."""
 
 import pytest
-
 
 from spond.spond import Spond
 
Skipped 2 files
```

## Actual behaviour:

Output does not include diffs. Extract from GitHub actions log. Note `--diff` does appear:

```
Running isort --check-only --diff .
Error: ERROR: /home/runner/work/Spond/Spond/tests/test_spond.py Imports are incorrectly sorted and/or formatted.
Error: Process completed with exit code 1.
```

i.e. similar to output I get on running without the `--diff` flag at CLI:

```
isort  --check-only .
ERROR: C:\Users\[REDACTED]\code\elliot-100_Spond\tests\test_spond.py Imports are incorrectly sorted and/or formatted.
Skipped 2 files
```

Please let me know if any extra information is required.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Packages are installed into a virtualenv due to the fact that system-wide
   installs fail by default when run on newer versions of Python.
+- Print stderr to console when action fails.
 
 ## [1.1.0] - 2022-11-04
 

--- a/bin/run_isort
+++ b/bin/run_isort
@@ -4,7 +4,7 @@
 source "${HOME}/isort-venv/bin/activate"
 
 echo "Running isort $*"
-isort_result=$(isort "$@")
+isort_result=$(isort "$@" 2>&1)
 exit_code=$?
 
 # The isort output can be a multiline string. By default, GITHUB_OUTPUT expects
@@ -31,4 +31,5 @@ DELIMITER=$(
     echo "${DELIMITER}"
 } >> "${GITHUB_OUTPUT}"
 
+echo "${isort_result}"
 exit $exit_code

